src/utils/spp.ts

```diff
@@ -1,10 +1,20 @@
 import request, { CliRequestOptions } from '../request.js';
 
+export interface SppModel {
+  ConfidenceScore?: string;
+  Explanations?: string;
+  Schemas?: string;
+  UniqueId: string;
+  Publications?: any[];
+  ModelSettings?: string;
+  'odata.null'?: boolean;
```

Comment:
This property is not really part of an `SppModel`. I suggest we remove it from here. In the command when we check for `odata.null`, let's just cast the result to `any` in the if-statement.

`if ((result as any)['odata.null'] === true) { .... }`